src/main/java/org/prebid/server/handler/openrtb2/AmpHandler.java

```diff
@@ -276,7 +281,7 @@ private void handleResult(AsyncResult<Tuple3<BidResponse, AuctionContext, AmpRes
             errorMessages = Collections.emptyList();
 
             status = HttpResponseStatus.OK;
-            routingContext.response().headers().add(HttpUtil.CONTENT_TYPE_HEADER, HttpHeaderValues.APPLICATION_JSON);
+            enrichWithHeaders(routingContext);
```

Comment:
The question - should we respond with `HttpUtil.X_PREBID_HEADER` for errors?

Reply:
Nice catch! I think yes, because error response is a part of server behaviour, which depends on its version.